src/openms/source/FORMAT/HANDLERS/ConsensusXMLHandler.cpp

```diff
@@ -258,7 +258,7 @@ namespace OpenMS::Internal
             act_index_tuple.setCharge(charge);
           }
 
-          act_cons_element_.insert(std::move(act_index_tuple));
+          act_cons_element_.insert(act_index_tuple);
```

Comment:
also here... quick check might be good

Reply:
`ConsensusFeature::insert` are all missing a move version

Reply:
will address this is in #5685